project_event/report/project_task_activity_report.py

```diff
@@ -53,6 +53,7 @@ def get_task_values(self, tasks):
                     'department': task.department_id.name,
                     'expected_start': self.get_tz_format(task.date_start),
                     'employee': employee.name,
+                    'resource': task.equipment_id.name,
```

Comment:
This could be None. So None.name would cause a problem

Reply:
Added a condition to display the equipment name otherwise a text message Not a service is displayed instead.